perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/RegistrarManagerMethod.java

```diff
@@ -479,14 +498,33 @@ public ApplicationFormItem call(ApiCaller ac, Deserializer parms) throws PerunEx
 	 * @param state List<String> List of states: NEW, VERIFIED, APPROVED, REJECTED
 	 * @return List<Application> Found applications
 	 */
+	/*#
+	 * Gets all applications in a given state for a given Group in a given date period.
+	 *
+	 * @param group int Group <code>id</code>
+	 * @param state List<String> List of states: NEW, VERIFIED, APPROVED, REJECTED
+	 * @param dateFrom String Earliest date for applications
+	 * @param dateTo String Latest date for applications
+	 * @return List<Application> Found applications
+	 */
 	getApplicationsForGroup {
 
 		@Override
 		public List<Application> call(ApiCaller ac, Deserializer parms) throws PerunException {
-			if (parms.contains("state")) {
-				return ac.getRegistrarManager().getApplicationsForGroup(ac.getSession(), ac.getGroupById(parms.readInt("group")), parms.readList("state", String.class));
+			if (parms.contains("dateFrom")) {
```

Comment:
This way, you can specify only one bound.
```suggestion
			if (parms.contains("dateFrom") || parms.contains("dateTo")) {
```

Reply:
Added.